src/types/models/tokens.ts

```diff
@@ -59,9 +59,20 @@ type CreateToken<DataType = Primitive> = Pick<
   deduplicateToken?: boolean;
 };
 
+type UpdateToken<DataType = Primitive> = Partial<
+  Pick<
+    Token<DataType>,
+    'data' | 'metadata' | 'searchIndexes' | 'fingerprintExpression'
+  > & {
+    privacy: Omit<TokenPrivacy, 'classification'>;
```

Comment:
```suggestion
    privacy?: Omit<TokenPrivacy, 'classification'>;
```
Wouldn't this be optional?

Reply:
Everything is optional AFAIK, have the object wrapped on a `Partial`

Reply:
I took off the other ? from that object to avoid confusion

Reply:
oh yeah. `Partial` should do the job